company-profiles/darecode.md

```diff
@@ -0,0 +1,33 @@
+# darecode
+
+## Company blurb
+
+At [dareCode](https://dareCode.com) our mission is to help companies to transform IT by doing things in other ways: 100% tecky and transparent. We work with worldwide enterprise companies and best partners. We are OpenSource supporters, DevOps enthusiast and software developers.
+
+## Company size
+
+We’re 50 ppl in 2019 and aim to growth safely to reach 75 team members in y2020.
+
+## Remote status
+
+We have offices in Spain (Madrid and Zaragoza) and several remote professionals in several cities. We work with local and remote members.
+
+## Region
+
+Worldwide but specially European market. Our customers are in several locations.
```

Comment:
Typo: "specially" &rarr; "especially" (this is one of the few words that start with "es" in both English and Spanish!)

Reply:
Thank you @nylen! you are right

Fixed

Best regards